linalg/linalg.hpp

```diff
@@ -32,6 +32,9 @@
 #include "invariants.hpp"
 #include "constraints.hpp"
 #include "auxiliary.hpp"
+#include "batched/batched.hpp"
+#include "batched/gpu_blas.hpp"
```

Comment:
I think this means that .hpp files will now have to be compiled with device compilers whenever users add mfem.hpp into their projects? I think this may effect us. I will test....

Reply:
If `MFEM_USE_CUDA` is not defined, then `gpu_blas.hpp` won't include any CUDA headers.

If there is a problem with this in your application then we can change things a bit so that the CUDA headers are needed only in the cpp files and not the headers. (They are only needed in the headers so that `GPUBlas::Handle()` can return the correct type (`cublasHandle_t` or `hipblasHandle_t`) — this could be useful for those wanting to use cuBLAS/hipBLAS more generally, which is why it's available in the header).